test/definitions-eu/apm/dotNet/windows/windows2016-iis.json

```diff
@@ -44,20 +44,21 @@
         "is_core_app": "true"
       }
     }],
-  
-    "instrumentations": {
-      "resources": [
-        {
-            "id": "nr_infra",
-            "resource_ids": ["iiswindows2016"],
-            "provider": "newrelic",
-            "source_repository": "https://github.com/newrelic/open-install-library.git",
-            "deploy_script_path": "test/deploy/windows/newrelic-cli/install/roles",
-            "params": {
-                "local_recipes": true
-            }
+
+  "instrumentations": {
+    "resources": [
+      {
+        "id": "nr_infra",
+        "resource_ids": ["iiswindows2016"],
+        "provider": "newrelic",
+        "source_repository": "https://github.com/newrelic/open-install-library.git",
+        "deploy_script_path": "test/deploy/windows/newrelic-cli/install-recipe/roles",
+        "params": {
+          "recipe_content_url": "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/infrastructure/windows.yml,https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/apm/dotNet/windows-iis.yml",
```

Comment:
[What is the reason behind instrumenting .net here, as far as I know. IIS is a microsoft web server. How are you making sure a .net application is running it. Does above services will make sure .net application is running? Just want to check this with you](https://github.com/newrelic/open-install-library/pull/1135/files#r1855507409)

Reply:
The reason for instrumenting .NET here is to ensure that the .NET application running on the IIS web server is properly monitored. IIS is a Microsoft web server commonly used to host .NET applications. The services defined will set up the IIS server and deploy the .NET application, while the instrumentation ensures that the application is monitored by New Relic. The validate_output parameter confirms the .NET agent installation, ensuring the application is running. After adding these changes I am able to deploy successfully.